spec/graphql/static_validation/rules/fragments_are_used_spec.rb

```diff
@@ -4,7 +4,7 @@
   include StaticValidationHelpers
   let(:query_string) {"
     query getCheese {
-      name,
+      name
```

Comment:
No, just a typo I noticed. `,` aren't required here.

I could break this up into a separate commit if you prefer.